pkg/loki/filter.go

```diff
@@ -141,7 +142,7 @@ func (f *lineFilter) asLabelFilters() []labelFilter {
 			valueType: v.valueType,
 			value:     v.value,
 		}
-		if v.valueType == typeRegex || v.valueType == typeRegexContains {
+		if v.valueType == typeRegex || v.valueType == typeRegexContains || v.valueType == typeRegexArrayContains {
```

Comment:
maybe worth small helper to check for all regex flavor ?

Reply:
sure !